inst/rmd/h/pandoc/elevate-section-attrs.js

```diff
@@ -0,0 +1,16 @@
+$(function() {
+  $("div[class*='section level']").each(function(i, el) {
```

Comment:
Is this different/better than `"div.section.level"`?

Reply:
Yea, `level` has a suffix reflecting the level of the section (e.g., `level1`, `level2`, etc).

This selector matches any div where `class` contains `'section level'`. I'm not 100% sure the ordering of those classes always appears to be case for the versions of pandoc we're applying this patch to

Reply:
`div.section[class*='level']`?

Reply:
Sort of related to this concern...should the selector be `div[class*='level'], section[class*='level']` so that we're also compatible with `html5`?

```shell
pandoc -t html5 --section-divs <<< "# Hello {.foo}" 
```

```html
<section id="hello" class="level1">
<h1 class="foo">Hello</h1>
</section>
```

Reply:
We probably won't move to `html5` in the foreseeable future. It will break a lot of things (we'll have to bite the bullet one more time someday). Anyway, it is harmless to add the `section` selector here.